src/idaapi.py

```diff
@@ -16,68 +17,88 @@
 PLUGIN_OK = 1
 SETMENU_APP = 1
 
+# uefi re tool
+PLUGIN_MOD = 1
+PLUGIN_PROC = 64
+PLUGIN_FIX = 128
+BADADDR = hex(4294967295)
```

Comment:
???

This is not well implemented.
Consider defining the EA type (effective address) as (and, BTW start adding types)
```python
class BADADDR:
    pass

EA = int | BADADDR
```

Also, there is another definition of BADADDR as -1 (which may or may not numerically match). Unify the definitions, decide where to store the types and constants in terms of who uses them. Design the layering and "X-uses-Y" relationships without cyclic imports.